lib/qbwc/producer.rb

```diff
@@ -44,11 +44,9 @@ def build_polling_request
         params = record.values.first
 
         klass = "QBWC::Request::#{object_type.capitalize}".constantize
-        string << klass.polling_others_items_xml(params['quickbooks_since'], @config)
 
-        params_to_send = params['quickbooks_since']
-        params_to_send = params if params['return_all']
-        string << klass.polling_current_items_xml(params_to_send, @config)
+        string << klass.polling_others_items_xml(params['quickbooks_since'], @config)
```

Comment:
As a side note, I thought it was interesting that only one class , `Inventories`,  implements `#polling_others_items_xml` with more than an empty string. That class might be a good first candidate to pull out from the metaprogramming parts in the future

Reply:
Agreed - I saw the same thing haha